force-app/main/default/aura/AuraCalculator/AuraCalculator.cmp

```diff
@@ -1,9 +1,5 @@
 <aura:component controller="AuraControllerCalc"  implements="force:appHostable,flexipage:availableForAllPageTypes,flexipage:availableForRecordHome,force:hasRecordId,force:lightningQuickAction" access="global" >
-    <aura:attribute name="label" type="String" default=""/>
-    <aura:registerEvent name="DisplayVaueEvent" type="c:ButtonDisplayValue"/>
-   
-    <lightning:button label="{!v.label}" name="{!v.label}" onclick = "{!c.press}"></lightning:button>
-    
-    
-    
+  <aura:attribute name="label" type="String" default=""/>
+  <aura:registerEvent name="DisplayVaueEvent" type="c:ButtonDisplayValue"/>
```

Comment:
Please correct the spelling of the name of the event. It should be DisplayValueEvent.